tools/oesign/oesign.c

```diff
@@ -378,6 +465,26 @@ void _merge_config_file_options(
     if (options->security_version.has_value)
         properties->config.security_version = options->security_version.value;
 
+    bool kss_supported = _is_kss_supported();
+    if (kss_supported)
+    {
+        if (options->family_id.has_value)
+            memcpy(
+                properties->config.family_id,
+                &options->family_id.value,
+                sizeof(options->family_id.value));
+
+        if (options->extended_product_id.has_value)
+            memcpy(
+                properties->config.extended_product_id,
+                &options->extended_product_id.value,
+                sizeof(options->extended_product_id.value));
+
+        if (options->family_id.has_value ||
+            options->extended_product_id.has_value)
+            properties->config.attributes |= OE_SGX_FLAGS_KSS;
```

Comment:
Even though the 2 definitions are identical, you should use SGX_FLAGS_KSS instead of OE_SGX_FLAGS_KSS to be consistent with other attributes assignments in this file.

Reply:
Updated